reward_extraction/behavior_cloning/behavior_cloning.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
```

Comment:
maybe move this into a new folder called bc at the root of the repo? since bc isn't really related to reward extraction, it's just supervised learning

Reply:
That makes sense, we weren't sure where to create a new folder so we just temporarily put it in rewad_extraction/. Just moved it to the root, thanks